app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -0,0 +1,11 @@
+
+attributes :content_view_id, :content_view_name
+attributes :lifecycle_environment_id, :lifecycle_environment_name
+attributes :uuid
+
+node :errata_counts do |content_facet|
```

Comment:
I suppose not for now, things can be added in the future if users decide. I was more curious than anything.